### DISCUSSION TITLE
devicetime

### DISCUSSION THREAD

**Discussion by papax333** [softwarecrash/EPEver2MQTT#34] [General]
Hello,
Is it possible to set the device time automatically at a defined time interval? For example, daily at 10.00 am.
Setting via MQTT would also be conceivable, if possible.

**Comment by softwarecrash** [comment]
hello,
actual not

**Comment by papax333** [comment]
ok, I took a look at the source code and realized that it is quite easy to do. You can simply use an http call with http://xxx.xxx.xxx.xxx/set?datetime=YYMMDDSSmmss

**Comment by softwarecrash** [comment]
yes, i forgot that it exsist a webhook like that.
